flexget/plugins/sites/nyaa.py

```diff
@@ -68,11 +93,21 @@ def search(self, task, entry, config):
                 entry = Entry()
                 entry['title'] = item.title
                 entry['url'] = item.link
-                # TODO: parse some shit
-                # entry['torrent_seeds'] = int(item.seeds)
-                # entry['torrent_leeches'] = int(item.leechs)
-                # entry['search_sort'] = torrent_availability(entry['torrent_seeds'], entry['torrent_leeches'])
-                # entry['content_size'] = int(item.size) / 1024 / 1024
+                entry['torrent_seeds'] = int(item.nyaa_seeders)
+                entry['torrent_leeches'] = int(item.nyaa_leechers)
+                entry['torrent_info_hash'] = item.nyaa_infohash
+                entry['search_sort'] = torrent_availability(entry['torrent_seeds'], entry['torrent_leeches'])
+                nyaa_size_split = item.nyaa_size.split(' ')
+                if len(nyaa_size_split):
+                    size_multiplier = float(1.0 / 1024.0**2)  # Assume it is in bytes
```

Comment:
https://github.com/Flexget/Flexget/blob/develop/flexget/utils/tools.py#L442

Reply:
Oh! That's neat. Let's use that.